docs/colors.md

```diff
@@ -31,6 +31,25 @@ React Native supports also colors as an `int` values (in RGB color mode):
 
 > **_Note:_** This might appear similar to the Android [Color](https://developer.android.com/reference/android/graphics/Color) ints representation but on Android values are stored in SRGB color mode (0xaarrggbb).
 
+## DynamicColorIOS
+
+```jsx
+DynamicColorIOS({light: [Object object], dark: [Object object]}): [Object object]
```

Comment:
`[Object object]` probably isn't super great for this since it could be "red", or "rgb(128,255,255)". How else could we make this signature seem like it supports any color for light and dark?

Also, just like PlatformColor, the return type isn't needed

Reply:
Perhaps merely 
```
DynamicColorIOS({light: color, dark: color})
```
?